examples/window-app/common/include/ExampleApp.h

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
```

Comment:
nit: here and most of the other new files, Set copyright Copyright (c) 2021 Project CHIP Authors

Reply:
Copyright updated on the new files.